src/vim.h

```diff
@@ -1433,6 +1433,7 @@ typedef enum
     , HLF_CUC	    /* 'cursurcolumn' */
     , HLF_CUL	    /* 'cursurline' */
     , HLF_MC	    /* 'colorcolumn' */
+    , HLF_QFC	    /* 'quickfix window current line' */
```

Comment:
The single quotes are meant to be around the Vim options  (such as 'colorcolumn', 'cursorline', etc.), not around the full comment. Here it's not a vim option, so don't put the single quote.